gaas-ui/src/setupProxy.js

```diff
@@ -7,32 +7,25 @@ const { createProxyMiddleware } = require("http-proxy-middleware");
  */
 module.exports = function (app) {
     app.use(
-        "/graphs",
+        '/graphs',
         createProxyMiddleware({
-            target: "http://localhost:4000",
+            target: 'http://localhost:4000',
             changeOrigin: true,
         })
     );
 
     app.use(
-        "/namespaces",
+        '/namespaces',
         createProxyMiddleware({
-            target: "http://localhost:4000",
-            changeOrigin: true,
-        })
-    );
-    app.use(
-        "/storetypes",
-        createProxyMiddleware({
-            target: "http://localhost:4000",
+            target: 'http://localhost:4000',
             changeOrigin: true,
         })
     );
 
     app.use(
-        "/auth",
```

Comment:
Just wondering why this got changed to single quotes?

Reply:
linting has been fixed on this file